### PR TITLE
fix ghosting and stuck screens in macOS

### PR DESCRIPTION
**Pull request by vitor-jotave**
enabling hardware acceleration fixed ghosting graphics and stuck screens in macOS for me, may help other mac players. 

(MacBook Pro, Sonoma, M2 Pro)

before enabling: https://drive.google.com/file/d/1PEqsYOH-KF3XNVdCYwIG9AoM1mu5gszL/view?usp=sharing

after enabling: https://drive.google.com/file/d/1AT8YLfjifYwG1i2UPa5WVVD_TwpJ7INo/view?usp=sharing

### CODE REVIEW

**Comment by SnosMe** [comment]
That's... interesting.

File: main/src/main.ts [hunk 1]
```diff
@@ -19,7 +19,6 @@ if (!app.requestSingleInstanceLock()) {
   app.exit()
 }
 
-app.disableHardwareAcceleration()
```
Comment (SnosMe) [inline main/src/main.ts:22]: if (platform != macOS) {} please

**Review by SnosMe — CHANGES_REQUESTED** [review]

**Comment by vitor-jotave** [comment]
there it is :)

File: main/src/main.ts (hunk 1 above)
Comment (vitor-jotave) [reply main/src/main.ts:22]: done

**Comment by davidtubehd** [comment]
how do u enable hardware acceleration?

**Comment by marconett** [comment]
can confirm that this patch fixes the issue.

**Comment by marconett** [comment]
or rather: removing`app.disableHardwareAcceleration()` fixes the issue. the patch itself has syntax errors ..

**Comment by anstaendig** [comment]
Any chance this will PR will get merged? :)

**Comment by SnosMe** [comment]
Yes, it will ve

**Comment by flieee** [comment]
[machwaccel.patch](https://github.com/user-attachments/files/16571158/machwaccel.patch)

Hi @SnosMe, @vitor-jotave, would this patch work?

**Comment by vitor-jotave** [comment]
> [machwaccel.patch](https://github.com/user-attachments/files/16571158/machwaccel.patch)
> 
> Hi @SnosMe, @vitor-jotave, would this patch work?

seems like it should, up to @SnosMe though.

**Pull request closed without merging**